spec/example_app/app/assets/config/manifest.js

```diff
@@ -1,2 +1,4 @@
-//= link administrate/application.css
-//= link administrate/application.js
+//= link administrate-internal/docs.css
+
+//= link_tree ../images
+//= link_tree ../builds
```

Comment:
@pablobm  I've restored this line. I think it is necessary to link `../builds` here. I'll check again